src/ProjectTemplates/test/Templates.Tests/template-baselines.json

```diff
@@ -1698,6 +1698,8 @@
         "Pages/Index.razor",
         "Properties/launchSettings.json",
         "wwwroot/css/app.css",
+        "wwwroot/icon-192.png",
+        "wwwroot/icon-512.png",
```

Comment:
Only templates created with the `--pwa` flag should have these icon files. There was probably some template.json setting we had before that handled this filtering which may've been removed when these icons were originally removed. 

Also, do we need both icons? I believe just one may be sufficient.

Reply:
Added this setting back for the icon files to be excluded if the `--pwa` flag isn't specified. The `icon-192` file has been removed.